docs/docs/mapping/customizing_openapi_output.md

```diff
@@ -179,31 +176,32 @@ The content of `tables.md`:
 
 ```markdown
 ## {{.RequestType.Name}}
-| Field ID    | Name      | Type                                                       | Description                  |
-| ----------- | --------- | ---------------------------------------------------------  | ---------------------------- | {{range .RequestType.Fields}}
-| {{.Number}} | {{.Name}} | {{if eq .Label.String "LABEL_REPEATED"}}[]{{end}}{{.Type}} | {{fieldcomments .Message .}} | {{end}}  
- 
+
+| Field ID | Name | Type | Description |
+| ----------- | --------- | --------------------------------------------------------- | ---------------------------- | {{range .RequestType.Fields}}
+| {{.Number}} | {{.Name}} | {{if eq .Label.String "LABEL_REPEATED"}}[]{{end}}{{.Type}} | {{fieldcomments .Message .}} | {{end}}
+
 ## {{.ResponseType.Name}}
-| Field ID    | Name      | Type                                                       | Description                  |
+
+| Field ID | Name | Type | Description |
 | ----------- | --------- | ---------------------------------------------------------- | ---------------------------- | {{range .ResponseType.Fields}}
-| {{.Number}} | {{.Name}} | {{if eq .Label.String "LABEL_REPEATED"}}[]{{end}}{{.Type}} | {{fieldcomments .Message .}} | {{end}}  
+| {{.Number}} | {{.Name}} | {{if eq .Label.String "LABEL_REPEATED"}}[]{{end}}{{.Type}} | {{fieldcomments .Message .}} | {{end}}
```

Comment:
Is this still formatted correctly? I remember us talking about an auto-formatter potentially breaking these tables.